### PR TITLE
Fix Potential Problem in MinedTxHandler

### PR DESCRIPTION
**Pull request by chgorman**
## Scope

Fixing a possible bug in MinedTxHandler that does not allow for the correct use of HeightIdx indexer.

## Why?

This will allow for correct utilization of the HeightIdx indexer.

### CODE REVIEW

File: application/minedtx/mined_test.go [hunk 1]
```diff
@@ -274,6 +317,135 @@ func TestMined(t *testing.T) {
 	}
 }
 
+func TestMined2(t *testing.T) {
```
Comment (diligentis) [inline application/minedtx/mined_test.go:320]: Can we get a more descriptive name here?

File: application/minedtx/mined_test.go [hunk 2]
```diff
@@ -134,6 +134,49 @@ func makeTxInitial(t *testing.T, ownerSigner objs.Signer) (objs.Vout, *objs.Tx)
 	return consumedUTXOs, tx
 }
 
+func makeTxInitial2(t *testing.T, ownerSigner objs.Signer, numTxIns uint32) (objs.Vout, *objs.Tx) {
```
Comment (diligentis) [inline application/minedtx/mined_test.go:137]: How does this differ from `makeTxInitial`?

**Review by diligentis — COMMENTED** [review]
Have some questions

File: application/minedtx/mined_test.go (hunk 1 above)
Comment (chgorman) [reply application/minedtx/mined_test.go:320]: I will use this one to overwrite the previous `TestMined` test.

File: application/minedtx/mined_test.go (hunk 2 above)
Comment (chgorman) [reply application/minedtx/mined_test.go:137]: It allows for specifying the number of TxIns. I will use this version and remove the other.

**Pull request merged**